custom/icds_reports/reports/disha.py

```diff
@@ -28,8 +29,18 @@ def __init__(self, state_name, month):
     def _blob_id(self):
         return 'disha_dump-{}-{}.json'.format(self.state_name.replace(" ", ""), self.month.strftime('%Y-%m-%d'))
 
-    def get_data(self):
-        dump = IcdsFile.objects.filter(blob_id=self._blob_id()).first()
+    @memoized
+    def _get_file_ref(self):
+        return IcdsFile.objects.filter(blob_id=self._blob_id()).first()
```

Comment:
So these files stay on the server for each month and state? forever?

Reply:
There is a an expiry period of a year.

Reply:
so thats
number of states * 12 files in total.
Do we need to worry about the storage it would consume?